bin/triton-mysql.py

```diff
@@ -205,13 +205,13 @@ def put_backup(self, backup_id, infile):
 
 class Containerpilot(object):
     """
-    Containerpilot config is where we rewrite Containerpilot's own config
+    Containerpilot config is where we rewrite ContainerPilot's own config
```

Comment:
I think I correctly capitalized ContainerPilot in all the text, but left it as `Containerpilot` when referencing the class.